### ISSUE TITLE
Improve attempted Northern Ireland swap attempts

### ISSUE THREAD

**Issue by tdg** [swapmyvote/swapmyvote#176]
Initially (M1) - don't show polls for NI parties

Use the space instead to say 

“Northern Ireland has a largely different set of political parties than the rest of the UK. 
Swap My Vote doesn’t currently support swaps there, but we are working on this for future elections”

**Comment by tdg** [comment]
We need to think about and take advice on any political/psychological/trust issues involved in swaps between NI and GB voters for future elections. 

Supporting NI-NI vote swaps might be a useful interim goal, but we have noted no public demand from NI at all in 2015/7 elections.

**Comment by aspiers** [comment]
This is a very ugly frontend bug.  It should in theory be fairly easy to replace the error with some apology text - what do you think @baob ?

**Comment by baob** [comment]
Yes, no matter how ugly the quick fix apology, it has to be better than the error message... and it should be relatively easy.

From my iPhone
(Please forgive my typos and the phones inappropriate auto-corrections)

**Comment by baob** [comment]
Snapshot of the bug.
![image](https://user-images.githubusercontent.com/55932/69754278-6ef62300-114d-11ea-9917-3d56c5d5113f.png)

**Comment by aspiers** [comment]
@baob Is this still a problem with the new data?

**Comment by baob** [comment]
> Is this still a problem with the new data?

Yes. There is still an issue with NI constituencies ... no polling data, and their main parties are not (yet) included in our parties dropdown.

BUT

* One of the TV sites offers recommendations (which we don't import because we have no matching parties). 
* the democracy club candidates data will tell us which parties to include. 

In the mean time, NI constituencies look like this 

![image](https://github.com/swapmyvote/swapmyvote/assets/55932/b77504ab-4c53-42f8-a610-fc203a53c0ff)

**Comment by aspiers** [comment]
Well, that's not too terrible so I think we can live with it if need be :sweat_smile:
